server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -103,7 +104,7 @@ public CompletableFuture<Collection<Token>> listTokens(User loginUser) {
     @StatusCode(201)
     @ResponseConverter(CreateApiResponseConverter.class)
     public CompletableFuture<HttpResult<Token>> createToken(@Param String appId,
-                                                            @Param boolean isAdmin,
+                                                            @Param @Default("false") boolean isAdmin,
```

Comment:
Question) Is this related to changes in this PR? Or is this a separate improvement?

Reply:
It is not related to this PR.
I've fixed it in this PR together since it looked like a trivial change.